src/applications/personalization/profile360/actions/updaters.js

```diff
@@ -46,7 +46,7 @@ export function refreshTransaction(transaction, analyticsSectionName) {
         transaction
       });
 
-      const transactionRefreshed = isVet360Configured() ? await apiRequest(`/profile/status/${transactionId}`) : await localVet360.updateTransactionRandom(transactionId);
+      const transactionRefreshed = isVet360Configured() ? await apiRequest(`/profile/status/${transactionId}`) : await localVet360.updateTransaction(transactionId);
 
```

Comment:
This is configured to succeed every time now so that E2E tests run consistently.